### ISSUE TITLE
support for laravel 10 ^10.0

### ISSUE THREAD

**Issue by donejeh** [plesk/ext-laravel-integration#6]
![image](https://github.com/plesk/ext-laravel-integration/assets/14345639/1ef4fbdb-dc15-4947-a914-d10a773e32ba)

i have issese enable queue on plesk with laravel 10

![image](https://github.com/plesk/ext-laravel-integration/assets/14345639/047b9601-420a-4507-8557-67cc3cf3e766)

**Comment by Askancy** [comment]
Did you manage to solve it?  I also have "Scheduled Tasks" disabled that you don't want to turn on....

**Comment by donejeh** [comment]
> "Scheduled Tasks"

You have to install Plesk extension to have that toggle button enable automatically

![image](https://github.com/plesk/ext-laravel-integration/assets/14345639/5c032110-9808-4b16-b5b1-12f6019886b7)

**Comment by rybalov** [comment]
That's right. In order for Laravel Toolkit to manage your queues, you need to add this dependency to your composer.json.
More information here: https://docs.plesk.com/en-US/obsidian/administrator-guide/website-management/laravel-toolkit.80010/#managing-queues

**Comment by Askancy** [comment]
@rybalov  The dependency is there and it is updated, are you sure it is compatible with Laravel 10?


![image](https://github.com/plesk/ext-laravel-integration/assets/789702/e4876cf4-b17a-49f5-8d15-f3134ffa3bd4)

**Comment by Askancy** [comment]
@rybalov So the ticket is closed without verification? Nice support though from Plesk...  😂😂

**Comment by rybalov** [comment]
@Askancy, sorry for our misunderstanding.

If you follow the points mentioned in the documentation, queue support with Laravel 10 works without problems.

Let's make sure that you have fulfilled all the requirements correctly. In addition to installing the component, you need to ensure the following:

1. This domain uses PHP >= v8.1 (Laravel 10 requirements).
2. Subscription permission "Scheduler management" must be enabled (either through customization or through a service plan).
<img width="556" alt="Screenshot 2023-10-11 at 19 17 18" src="https://github.com/plesk/ext-laravel-integration/assets/9481760/3036153e-026a-47b5-9607-9f7ed5dcd883">

3. The shell for this subscription must be selected (Domain -> Hosting & DNS -> Hosting -> SSH access).
<img width="603" alt="Screenshot 2023-10-11 at 19 17 39" src="https://github.com/plesk/ext-laravel-integration/assets/9481760/dc96ac01-d3fa-4257-be2e-6891a5911b82">

As a result, the queue switch is available for enabling.
<img width="341" alt="Screenshot 2023-10-11 at 19 18 14" src="https://github.com/plesk/ext-laravel-integration/assets/9481760/cd8ce1e7-70b1-4145-bce5-e96d92f419ba">


Let me know if the problem still occurs.

**Comment by Askancy** [comment]
![image](https://github.com/plesk/ext-laravel-integration/assets/789702/b7578438-1c1f-404b-bd0a-915a5d828e4b)

![image](https://github.com/plesk/ext-laravel-integration/assets/789702/c81aec55-68d5-4c66-b8b2-bb6fedfd16aa)

my Plesk package did not see subscriptions, however as you see from the screenshot it is enabled:

![image](https://github.com/plesk/ext-laravel-integration/assets/789702/f118015b-6c99-45f3-9cd7-772743f18cb6)

![image](https://github.com/plesk/ext-laravel-integration/assets/789702/fc7d2ea3-ec7d-44e3-81b6-3f70b790a782)

I had already followed all the guidelines, but it keeps getting disabled.
I will preface by saying that the queues on my Laravel work correctly. however having a panel on plesk would be handy.

**Comment by rybalov** [comment]
Please hover over "Disabilitate" ("Attività pianificate"). What message in tooltip is displayed?

**Comment by Askancy** [comment]
![image](https://github.com/plesk/ext-laravel-integration/assets/789702/5aa40d1f-2c47-46b9-b55c-235f42905322)

But scheduler management is active, in fact I have it in the list of "applications" in my domain.

**Comment by rybalov** [comment]
Indeed, a bug was found in the permissions subsystem that caused the Task Scheduler permissions to be defined incorrectly in some cases.

The fix will appear in the next update (early next week), your Plesk Panel will receive it automatically.

**Comment by Askancy** [comment]
Now the Scheduled Tasks turn out to be enabled the queues do not
once enabled though if I exit and re-enter plesk, the option is turned off, watch the video:
If I try to reactivate it, I get the error:
>The scheduled task already exists.

In the schduled tasks tab in fact I have the warning:
>Scheduled tasks are disabled.

https://github.com/plesk/ext-laravel-integration/assets/789702/c87fda46-02d5-4b1b-ae44-31b7c7f6e945

**Comment by rybalov** [comment]
Is the problem reproduced on a different domain?
Can you show a list of your Scheduled Tasks?

**Comment by Askancy** [comment]
>Is the problem reproduced on a different domain?

I have no way of trying...

>Can you show a list of your Scheduled Tasks?
![image](https://github.com/plesk/ext-laravel-integration/assets/789702/68f9b2f2-c10c-4c29-8b9d-6db502a4961a)

Now, however, the "scheduled Tasks" is active (perhaps an update?) But Queue continues to be inactive.

![image](https://github.com/plesk/ext-laravel-integration/assets/789702/8b7b2dd5-e52d-43a7-b61b-83ffcc9c09f2)

**Comment by rybalov** [comment]
Ok. Could you show please a list of your Scheduled Tasks in the Domain Card (not in Laravel Toolkit)?

**Comment by Askancy** [comment]
> Ok. Could you show please a list of your Scheduled Tasks in the Domain Card (not in Laravel Toolkit)?

This?

![image](https://github.com/plesk/ext-laravel-integration/assets/789702/aaac4728-11cc-420f-adf7-4fc18e13eedb)

**Comment by donejeh** [comment]
You can use my fork version for laravel 10 and 11 just follow the step

https://github.com/donejeh/ext-laravel-integration

`        "plesk/ext-laravel-integration": "*"
`


![image](https://github.com/user-attachments/assets/1db77629-b56a-430b-8861-f659b8d17e19)


![image](https://github.com/user-attachments/assets/54cc7d06-7dfb-4b18-b166-f5933918a945)


`  "minimum-stability": "dev",
    "prefer-stable": true,
    "repositories": [
        {
            "type": "vcs",
            "url": "https://github.com/donejeh/ext-laravel-integration"
        }
    ]`

**Comment by pstivan** [comment]
Hello,

This discussion is quite old. 

If the problem still remains, the suggestion is to submit a request to [support](https://support.plesk.com/hc/en-us/articles/12377663472791).
